data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -109,7 +187,7 @@ then
 		"${KUBE_APISERVER_OPERATOR_IMAGE}" \
 		/usr/bin/cluster-kube-apiserver-operator render \
 		--manifest-etcd-serving-ca=etcd-ca-bundle.crt \
-		--manifest-etcd-server-urls={{.EtcdCluster}} \
+		--manifest-etcd-server-urls=$(etcd-server-urls $ETCD_MEMBERS) \
```

Comment:
fix the logic in go that sets the EtcdCluster value. bash function will not be accepted.

Reply:
Removed in openshift-metal3/kni-installer#130 so it'll be gone when this PR is refreshed